.repo-metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "text-to-speech",
+  "name_pretty": "Google Cloud Text-to-Speech API",
```

Comment:
Let's drop the API perhaps? otherwise we end up with phrases like: 

Otherwise we get "Enable the Google Cloud Text-to-Speech API API."